sw/device/lib/testing/mock_mmio.h

```diff
@@ -141,6 +141,20 @@ Int ToInt(std::initializer_list<BitField> fields) {
  */
 inline internal::LittleEndianBytes LeInt(const char *bytes) { return {bytes}; }
 
+Int Inverted(const char *bytes) {
+  char *inverted_bytes = malloc(sizeof(Int));
+  Int ret_val;
+
+  for (int i = 0; i < sizeof(Int); i++) {
```

Comment:
Nit, but we prefer prefix increment: `++i`. Check out our style guide (though this particular fact is inherited from Google's C++ style guide) https://docs.opentitan.org/doc/rm/c_cpp_coding_style/.